lib/main.js

```diff
@@ -394,7 +394,7 @@ define(['./errors'], function(errors) {
     // for easy membership testing with the "in" keyword
     function setLike(list, fill /* default true*/) {
         if(fill === undefined) fill = true;
-        var setLike = {};
+        var setLike = Object.create(null);
```

Comment:
I never tried ufoJS on IE ... I'd already be happy if cutting edge IE supports it. Also, NodeJS is an environment for ufoJS, though since io.js is part of it, it makes pretty good progress. So, probably `for ... of ` is also there.